src/main/java/com/faforever/client/fa/ForgedAllianceService.java

```diff
@@ -120,10 +124,21 @@ private LaunchCommandBuilder defaultLaunchCommand() {
         .executableDecorator(forgedAlliancePrefs.getExecutableDecorator())
         .executable(getExecutablePath());
 
+    return addDebugger(baseCommandBuilder);
+  }
+
+  private LaunchCommandBuilder replayLaunchCommand() {
+    LaunchCommandBuilder baseCommandBuilder = LaunchCommandBuilder.create()
+        .executableDecorator(forgedAlliancePrefs.getExecutableDecorator())
+        .executable(getReplayExecutablePath());
+
+    return addDebugger(baseCommandBuilder);
```

Comment:
We don't need to add the debugger to replays

Reply:
Is this the FAF Debugger?

Reply:
I think it is

Reply:
Jip said it's sometimes nice to have it on replays